data-avro/src/main/java/com/linkedin/data/avro/util/AvroUtil.java

```diff
@@ -61,7 +67,7 @@ private static String jsonFromGenericRecord(
     writer.setSchema(record.getSchema());
     writer.write(record, jsonEncoder);
     jsonEncoder.flush();
-    return outputStream.toString();
```

Comment:
Why not use https://docs.oracle.com/javase/7/docs/api/java/io/ByteArrayOutputStream.html#toString(java.lang.String)?

Reply:
the argument to that is a string, which causes a charset lookup, whereas what i used uses an already-looked-up Charset object.
will change